anndata/_io/h5ad.py

```diff
@@ -30,6 +30,16 @@
 from .specs import read_elem, write_elem
 from anndata._warnings import OldFormatWarning
 
+
+__all__ = [
+    "read_elem",
+    "write_elem",
+    "read_h5ad",
+    "write_h5ad",
+    "read_dataset",
+]
+
+
```

Comment:
Why put this in a private module?

Reply:
It clarifies what’s imported from elsewhere. That’s the list of functions that `anndata._io.zarr` has in an equivalent way. But we can get rid of it if you prefer.

Reply:
he prefers